content/posts/2021/10/libplanet-0.11-0.19/index.eng.md

```diff
@@ -0,0 +1,251 @@
+changed, and as we had responded to deal with diverse problems on the mainnet
+of our first game <cite>[Nine Chronicles]</cite>, powered by Libplanet, we'd been
+unable to afford to keep up with the updates.  So let's focus on some of
+the key changes that have made in between the past releases.
+
+[^1]: Libplanet is a common library that abstracts out the implementation
+      details of decentralized multiplayer gaming such as peer-to-peer
+      networking and data synchronization.
+
+[Libplanet]: https://libplanet.io/
+[Nine Chronicles]: https://nine-chronicles.com/
+
+
+New Release Cycle
+-----------------
+
+Previously, Libplanet had no clear rules for its release cycle.  It was just
+simply giving a new version number to a collection of unreleased changes when
+there is a big API change or some time has passed since the last release.
+
+However, since the launch of the <cite>Nine Chronicles</cite>' mainnet last
+year, Libplanet has had to make frequent releases to fix various problems of
+the mainnet.  Currently both teams of Libplanet and <cite>Nine Chronicles</cite>
+in Planetarium work closely together to catch up each other's updates.  This
+release cycle of Libplanet is likely to continue for a while.
+
+Under that policy, there have been 9 minor releases in the last few months,
+from 0.11.0 released in March 2021 to 0.19.0 released this week:
+
+- [0.11.0] (March 30)
+- [0.12.0] (July 23)
+- [0.13.0] (July 28)
+- [0.14.0] (August 5)
+- [0.15.0] (August 18)
+- [0.16.0] (August 25)
+- [0.17.0] (September 28)
+- [0.18.0] (October 13)
+- [0.19.0] (October 27)
+
+[0.11.0]: https://github.com/planetarium/libplanet/releases/tag/0.11.0
+[0.12.0]: https://github.com/planetarium/libplanet/releases/tag/0.12.0
+[0.13.0]: https://github.com/planetarium/libplanet/releases/tag/0.13.0
+[0.14.0]: https://github.com/planetarium/libplanet/releases/tag/0.14.0
+[0.15.0]: https://github.com/planetarium/libplanet/releases/tag/0.15.0
+[0.16.0]: https://github.com/planetarium/libplanet/releases/tag/0.16.0
+[0.17.0]: https://github.com/planetarium/libplanet/releases/tag/0.17.0
+[0.18.0]: https://github.com/planetarium/libplanet/releases/tag/0.18.0
+[0.19.0]: https://github.com/planetarium/libplanet/releases/tag/0.19.0
+
+
+Protocol Versions
+-----------------
+
+*Added in version [0.11.0].*
+
+A protocol version is a property of a block that represents in which version of
+Libplanet used for mining.  It helps games powered by Libplanet to easily
+upgrade the Libplanet version, and on the Libplanet side, it allows us to
+add new functionalities without breaking backward compatibility with existing
+networks.
+
+API-wise, it is exposed through
+the [`Block<T>.ProtocolVersion` property][Block<T>.ProtocolVersion], which is
+a 32-bit integer.  Those numbers solely represent a version of the network
+protocol, which is distinct from a version of the library package (on NuGet
+Gallery).  According to Semantic Versioning, patch releases never increase
```

Comment:
```suggestion
Gallery).  In compliance with Semantic Versioning, patch releases will never increase
```

'According to Semantic Versioning' may sound like there are some rules in semantic versioning about protocol version, which isn't what we intend to say.